File: 2235-capitalize-the-title/capitalize-the-title.cpp
class Solution {
public:
    string capitalizeTitle(string title) {
        stringstream ss(title);
        string word, result = "";
        while(ss>>word){
            int len = word.length();
            for(auto &it:word){
                it = tolower(it);
            }
            if(len>=3){
                word[0] = toupper(word[0]);
            }
            result+=word + " ";
        }
        result.pop_back();
        return result;
    }
};

File: 2235-capitalize-the-title/capitalize-the-title.py
class Solution:
    def capitalizeTitle(self, title: str) -> str:
        words = title.split()
        result = []
        for word in words:
            word = word.lower()
            if len(word) >= 3:
                word = word[0].upper() + word[1:]
            result.append(word)
        return " ".join(result)
